initial-setup.spec

```diff
@@ -138,7 +138,7 @@ if [ $1 -gt 1 ] ; then
 fi
 
 %files -f %{name}.lang
-%doc README.rst
+%doc README.rst ChangeLog
```

Comment:
I wasn't aware that this is still some Fedora best practice, in times of web accessible git repos.. But sure! Indeed on my system there are 108 /usr/share/doc/*/ChangeLog files.